assets/queries/terraform/aws/s3_bucket_with_unsecured_cors_rule/test/positive4.tf

```diff
@@ -0,0 +1,21 @@
+module "s3_bucket" {
+  source = "terraform-aws-modules/s3-bucket/aws"
+  bucket = "s3-tf-example-versioning"
+  acl    = "private"
+  version = "0.0.1"
+
+  versioning_inputs = [
```

Comment:
It seems that `versioning_input`  does not exists for this module, maybe you mean versioning https://github.com/terraform-aws-modules/terraform-aws-s3-bucket/blob/master/variables.tf#L97